index.html

```diff
@@ -1 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Lato:wght@700;900&display=swap" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+    <script src="js/nav.js" defer></script>
+    <script src="js/programs.js" defer></script>
+    <script src="js/popup.js" defer></script>
+    <script src="js/extra.js" defer></script>
+    <title>De Royale Institute</title>
+</head>
+<body>
+    <!-- =======================================
+                Nav Bar
+    ======================================== -->
+    <section class="socials mob-none">
```

Comment:
- [ ] [OPTIONAL] Good job implementing the changes suggested by the previous reviewer. But I think there was a bit of misunderstanding, all the links which are not currently active should be using the dark blue color (i.e `#272a31`) except the `De Royale` button. That one should always be orange. 🧐️
On hover, the link should change color to orange. The previous reviewer was asking you to remove the borders around links that were showing when hovering over them. It would be great if you update it accordingly. 😎️